clients/js/example/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node 
+COPY package*.json ./
+RUN npm install
+
+COPY . .
```

Comment:
It looks like these two copies duplicate eachother?  Why not just do a `COPY . .` first here?